native-image failing on helidon-quickstart-mp

## Environment Details
* Helidon Version: 2.2.1-SNAPSHOT
* Helidon MP
* GraalVM version: graalvm-ce-java11-20.2.0
* OS: MacOS and Linux

----------

## Problem Description
native-image build is failing for helidon-quickstart-mp (it works for SE):

```
[INFO] [helidon-quickstart-mp:14148]     (clinit):     900.53 ms,  4.51 GB
[INFO] [helidon-quickstart-mp:14148]   (typeflow):  28,645.66 ms,  4.51 GB
[INFO] [helidon-quickstart-mp:14148]    (objects):  41,128.92 ms,  4.51 GB
[INFO] [helidon-quickstart-mp:14148]   (features):   3,984.28 ms,  4.51 GB
[INFO] [helidon-quickstart-mp:14148]     analysis:  76,912.59 ms,  4.51 GB
[WARNING] Error: No instances of sun.security.provider.NativePRNG are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use -H:+TraceClassInitialization.
[WARNING] Detailed message:
[WARNING] Trace: Object was reached by
[WARNING] 	reading field java.security.SecureRandom.secureRandomSpi of
[WARNING] 		constant java.security.SecureRandom@2ad9a439 reached by
[WARNING] 	reading field sun.security.ssl.SSLContextImpl.secureRandom of
[WARNING] 		constant sun.security.ssl.SSLContextImpl$DefaultSSLContext@26f6fa75 reached by
[WARNING] 	reading field sun.security.ssl.SSLSocketFactoryImpl.context of
[WARNING] 		constant sun.security.ssl.SSLSocketFactoryImpl@1dafd2c reached by
[WARNING] 	reading field javax.net.ssl.HttpsURLConnection.defaultSSLSocketFactory
[WARNING]
[WARNING] com.oracle.svm.core.util.UserError$UserException: No instances of sun.security.provider.NativePRNG are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use -H:+TraceClassInitialization.
[WARNING] Detailed message:
[WARNING] Trace: Object was reached by
[WARNING] 	reading field java.security.SecureRandom.secureRandomSpi of
[WARNING] 		constant java.security.SecureRandom@2ad9a439 reached by
[WARNING] 	reading field sun.security.ssl.SSLContextImpl.secureRandom of
[WARNING] 		constant sun.security.ssl.SSLContextImpl$DefaultSSLContext@26f6fa75 reached by
[WARNING] 	reading field sun.security.ssl.SSLSocketFactoryImpl.context of
[WARNING] 		constant sun.security.ssl.SSLSocketFactoryImpl@1dafd2c reached by
[WARNING] 	reading field javax.net.ssl.HttpsURLConnection.defaultSSLSocketFactory
[WARNING]
[WARNING] 	at com.oracle.svm.core.util.UserError.abort(UserError.java:79)
[WARNING] 	at com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:217)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:765)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:555)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.lambda$run$0(NativeImageGenerator.java:468)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1407)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
[WARNING] Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: No instances of sun.security.provider.NativePRNG are allowed in the image heap as this class should be initialized at image runtime. To see how this object got instantiated use -H:+TraceClassInitialization.
[WARNING] Detailed message:
[WARNING] Trace: Object was reached by
[WARNING] 	reading field java.security.SecureRandom.secureRandomSpi of
  ARNING] 		constant java.security.SecureRandom@2ad9a439 reached by
[WARNING] 	reading field sun.security.ssl.SSLContextImpl.secureRandom of
[WARNING] 		constant sun.security.ssl.SSLContextImpl$DefaultSSLContext@26f6fa75 reached by
[WARNING] 	reading field sun.security.ssl.SSLSocketFactoryImpl.context of
[WARNING] 		constant sun.security.ssl.SSLSocketFactoryImpl@1dafd2c reached by
[WARNING] 	reading field javax.net.ssl.HttpsURLConnection.defaultSSLSocketFactory
[WARNING]
[WARNING] 	at com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:126)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:762)
[WARNING] 	... 8 more
[WARNING] Error: Image build request failed with exit status 1

```

## Steps to reproduce
1. Run the archetype: 
```
mvn -U archetype:generate -DinteractiveMode=false \
    -DarchetypeGroupId=io.helidon.archetypes \
    -DarchetypeArtifactId=helidon-quickstart-mp \
    -DarchetypeVersion=2.2.1-SNAPSHOT \
    -DgroupId=io.helidon.examples \
    -DartifactId=helidon-quickstart-mp \
    -Dpackage=io.helidon.examples.quickstart.mp
```

2. Buidl the project with `mvn clean  package -Pnative-image`

--- a/jersey/client/src/main/resources/META-INF/native-image/io.helidon.jersey/helidon-jersey-client/native-image.properties
+++ b/jersey/client/src/main/resources/META-INF/native-image/io.helidon.jersey/helidon-jersey-client/native-image.properties
@@ -1,0 +1,17 @@
+#
+# Copyright (c) 2021 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Args=--initialize-at-run-time=org.glassfish.jersey.client.internal.HttpUrlConnector

